.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+
+    steps:
+    
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+                ${{ runner.os }}-maven-
+      - name: Build with Maven
+        run: mvn clean install -DskipTests -T4 2>&1| grep -e 'Building GeoMesa' -e '\(maven-surefire-plugin\|maven-jar-plugin\|scala-maven-plugin.*:compile\)'
```

Comment:
we could use build/mvn and possibly cache the downloads of zinc if that's easy

Reply:
@elahrvivaz implemented. I've seen that caching is not mega fast, while the zinc download is. I'm going to investigate this w/o the cached zinc download